From 6c1c578a4cbeda5d515c89f5e6bcfb948d93ea1c Mon Sep 17 00:00:00 2001
From: sanchitintel <sanchit.jain@intel.com>
Date: Mon, 10 Oct 2022 00:10:39 +0000
Subject: [PATCH 35/36] Fix Windows build (#72)

__declspec(dllexport) should be present in the declaration as well, if it's present in the definition
---
 aten/src/ATen/native/quantized/cpu/fbgemm_utils.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/aten/src/ATen/native/quantized/cpu/fbgemm_utils.cpp b/aten/src/ATen/native/quantized/cpu/fbgemm_utils.cpp
index 60a105c624..aa2d57c1aa 100644
--- a/aten/src/ATen/native/quantized/cpu/fbgemm_utils.cpp
+++ b/aten/src/ATen/native/quantized/cpu/fbgemm_utils.cpp
@@ -13,7 +13,11 @@
 #include <c10/util/accumulate.h>
 #include <torch/custom_class.h>
 
+#ifdef BUILD_JIT_QUANTIZATION_SAVE
+TORCH_API torch::class_<LinearPackedParamsBase> register_linear_params();
+#else
 torch::class_<LinearPackedParamsBase> register_linear_params();
+#endif
 torch::class_<EmbeddingPackedParamsBase> register_embedding_params();
 
 #ifdef USE_FBGEMM
-- 
2.25.1

